test/functional/Java8andUp/playlist.xml

```diff
@@ -1042,7 +1042,7 @@
 	<test>
 		<testCaseName>JCL_Test_JIT_Helper</testCaseName>
 		<command>$(JAVA_COMMAND) $(JVM_OPTIONS) --add-exports=java.base/com.ibm.jit=ALL-UNNAMED \
-	-Xbootclasspath/a:$(Q)$(TEST_RESROOT)$(D)TestResources.jar$(Q) \
+	--add-opens=java.base/com.ibm.jit=ALL-UNNAMED -Xbootclasspath/a:$(Q)$(TEST_RESROOT)$(D)TestResources.jar$(Q) \
```

Comment:
Are you sure this will work for Java 8? I suppose we can test via a build. Also while we're in the area can you rename this test case name to "JCL_Test_JITHelpers" to make it searchable since the test file is Test_JITHelpers.java and the class name is JITHelpers. I would like to be able to find this line in the future by grepping for the class name.

Reply:
This test is only for SE90 and up, SE80 has it's own test above. I don't think the `--add-opens` is necessary or is even recognized in Java 8